src/renderer/services/slippi/slippi.service.mock.ts

```diff
@@ -56,6 +57,11 @@ class MockSlippiBackendClient implements SlippiBackendService {
     await this.authService.updateDisplayName(name);
   }
 
+  @delayAndMaybeError(SHOULD_ERROR)
+  public async acceptRules() {
+    // Do nothing? Not sure how this should work
```

Comment:
just `// Do nothing` is fine since this is a server side action. just need to set the rulesAccepted value to 1 so it skips this step once we are done testing.